### DISCUSSION TITLE
Is it possible to download a markdown file of the report

### DISCUSSION THREAD

**Discussion by kiil** [assafelovic/gpt-researcher#294] [Q&A] [answered]
Sometimes I would like to further edit the report made by GPT Researcher.

It would be nice if there was a way to get **markdown** in addition to the pdf, html and plain text one can currently get.

Either just an option to copy as markdown or download as an .md file.

Is there a way to do this or is this a planned feature?

**Comment by assafelovic** [comment, marked as answer]
The report itself is presented as a markdown and then converted to PDF. You can simply use the gpt-researcher library as is which will return a markdown string.
